serverless.yaml

```diff
@@ -136,25 +137,27 @@ functions:
           type: dynamodb
           arn:
             'Fn::GetAtt':
-              - ResourceDynamoDBTable
+              - ResourceDynamoDBTableV2
               - StreamArn
           batchSize: 100
           maximumRetryAttempts: 3
           startingPosition: LATEST
 
 resources:
+  - Description: FHIR Work on AWS Example Server; v1.2

```

Comment:
It's kind of nice to display the version somewhere, But It's very likely that we'll forget to update this description for future releases. 

We need a mechanism to ensure this is kept in sync. Or just remove the version until we have one.

Also, shouldn't it be v2.0.0?

Reply:
Good call removing for now